### PR TITLE
PGP-165 adding macos and windows to ci.yml

### PR DESCRIPTION
**Pull request by armanreyes3**
need to run CLI sanity checks for macos and windows ci runners.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   ],
   "rules": {
     "indent": ["error", 4],
-    "linebreak-style": ["error", "unix"],
```
Comment (armanreyes3) [inline .eslintrc.json:14]: breaks on windows runner... we could switch to warn?

Comment (aduros) [reply .eslintrc.json:14]: > D:\a\playpass\playpass\jest.config.js
> Error:    1:30  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

I don't really understand why this is breaking, aren't our source files using unix line feeds? Do they get transformed when the repo is cloned on Windows?

Anyways, let's just delete the eslint rule for now.

**Review by aduros — CHANGES_REQUESTED** [review]
Thanks Arman! Can we use a build matrix instead of copy-pasting the CI steps for each platform? https://ncorti.com/blog/howto-github-actions-build-matrix

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -80,3 +80,35 @@ jobs:
               env:
                   AWS_ACCESS_KEY_ID: ${{ secrets.DOCS_AWS_ACCESS_KEY_ID }}
                   AWS_SECRET_ACCESS_KEY: ${{ secrets.DOCS_AWS_SECRET_ACCESS_KEY }}
+
+    verifyCLI:
+        strategy:
+            matrix:
+                os: [macos-latest, windows-latest]
```
Comment (aduros) [inline .github/workflows/ci.yml:87]: Should we add `ubuntu-latest` here and remove the other CLI check from the main test?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -80,3 +74,35 @@ jobs:
...
+              env:
+                  NODE_AUTH_TOKEN: ${{ secrets.PLAYCO_PACKAGES_TOKEN }}
+            - run: npm run build
+            - run: npm test
+
+            - name: CLI sanity check
+              run: |
+                  node ./dist/cjs/cli/index.js help
+                  cd templates/daily-level/
+                  npm i
```
Comment (aduros) [inline .github/workflows/ci.yml:104]:
```suggestion
                  npm ci
```

**Review by aduros — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 3 above)
Comment (aduros) [reply .github/workflows/ci.yml:104]: Minor but would speed builds up a bit

**Pull request merged**